README.md

```diff
@@ -55,50 +55,48 @@ The source code in this package is released under the Apache-2.0 License. For fu
 The [3rdparty-licenses.txt](3rd-party-licenses.txt) contains a list of other open source components included in this package.
 
 
-## Installation & Usage as a ROS package
-
-### Dependencies
+## Dependencies
 This package depends on the linear algebra library [Eigen], which can be installed by
 
     sudo apt-get install libeigen3-dev
 
-The [ROS] dependencies are listed in the package.xml and can be installed by
+The [ROS 1] or [ROS 2] dependencies are listed in the package.xml and can be installed by
 
     rosdep install steering_functions
 
+## Installation & Usage as a [ROS 1] or [ROS 2] package
 
 ### Building
 
-To build this package from source, clone it into your catkin workspace and compile it in *Release* mode according to
+To build this package from source, clone it into your workspace and compile it in *Release* mode according to
 
-    cd catkin_ws/src
+    cd my_ws/src
     git clone https://github.com/hbanzhaf/steering_functions.git
-    catkin build steering_functions -DCMAKE_BUILD_TYPE=Release
+    catkin build steering_functions -DCMAKE_BUILD_TYPE=Release  # [ROS 1]
+    colcon build --packages-up-to steering_functions --cmake-args -DCMAKE_BUILD_TYPE=Release  # [ROS 1] or [ROS 2]
 
-To launch a demo of the package, execute
+To launch a [ROS 1] demo of the package, execute
 
-    source catkin_ws/devel/setup.bash
+    source my_ws/devel/setup.bash
     roslaunch steering_functions steering_functions.launch
 
 
 ### Linking
 
-To link this library with another [ROS] package, add these lines to your package's CMakeLists.txt
+Add this package as dependency to your package's `package.xml`
```

Comment:
Just: "to your package.xml". We write package.xml in the section Dependencies also without apostrophes. I am flexible, we just have to be consistent.